google/auth/compute_engine/credentials.py

```diff
@@ -93,6 +93,15 @@ def refresh(self, request):
             raise exceptions.RefreshError(exc)
 
     @property
+    def service_account_email(self):
+        """The service account email.
+
+        .. Note: This is not guarenteed to be set until :meth`refresh` has been
```

Comment:
You are "guarenteed" to need a spellcheck here. Also do you need lowercase `note` for Sphinx or are the directives case-insensitive?